### ISSUE TITLE
Missing rule to configure quotes within attributes

### ISSUE THREAD

**Issue by avi12** [sveltejs/eslint-plugin-svelte#1213]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.

### What version of ESLint are you using?

9.26.0

### What version of `eslint-plugin-svelte` are you using?

3.5.1

### What did you do?

<details>
<summary>Configuration</summary>
<pre>
export default [
  rules: {
    "@stylistic/js/quotes": [
      "warn",
      "double",
      {
        allowTemplateLiterals: true,
        avoidEscape: true
      }
    ],
    "svelte/html-quotes",
      "error",
      {
        prefer: "double",
        dynamic: {
          quoted: true,
          avoidInvalidUnquotedInHTML: true
        }
      }
    }
  }
];
</pre>
</details>

```svelte
<div class="something {isSomething ? 'something1' : 'something2'}"></div>
```

### What did you expect to happen?

Leave it as
```svelte
<div class="something {isSomething ? 'something1' : 'something2'}"></div>
```

### What actually happened?

Breaks the markup
```svelte
<div class="something {isSomething ? "something1" : "something2"}"></div>
```

### Link to **GitHub Repo** with Minimal Reproducible Example

https://github.com/avi12/svelte-eslint-test

**Comment by ota-meshi** [comment]
> Breaks the markup

What broke? I think your `.svelte` render fine.

**Comment by avi12** [comment]
Huh, is it an error in my IDE then?
![Image](https://github.com/user-attachments/assets/97f561b6-56ce-4557-9685-ca74a9e3d588)

**Comment by JounQin** [comment]
That means your IDE is not compatible with Svelte compiler well enough.

That's said, I'd expect that we should use **valid html** whenever possible IMO.

**Comment by ota-meshi** [comment]
The rule is working fine so I'm closing this issue.
If you'd like to see an option added to the rule, please open a new issue with the specification so that someone can implement it.
